arkouda/numpy/sorting.py

```
@@ -282,3 +282,66 @@ def sort(
         args={"alg": algorithm.name, "array": pda, "axis": axis},
     )
     return create_pdarray(cast(str, repMsg))
+
+
+@typechecked
+def searchsorted(
+    a: pdarray, v: Union[int_scalars, float64, bigint, pdarray], side: Literal["left", "right"] = "left"
+) -> Union[int, pdarray]:
+    """
+    Find indices where elements should be inserted to maintain order.
+
+    Find the indices into a sorted array `a` such that, if the corresponding
+    elements in `v` were inserted before the indices, the order of `a` would be preserved.
+
+    Parameters
+    ----------
+    a : pdarray
+        1-D input array. Must be sorted in ascending order. `sorter` is not currently supported.
```

I think we should throw an error if someone tries this on a multi-dimensional pdarray.